bin/commands/start.js

```diff
@@ -22,12 +22,32 @@
 /* eslint-disable no-console */
 
 const
+  fs = require('fs'),
   rc = require('rc'),
   ColorOutput = require('./colorOutput');
 
 const
   params = rc('kuzzle');
 
+function loadJson (filePath) {
```

Comment:
I use this function because `require` take a path relative to the script directory and `fs` is relative from where you run the script.  
You can not do this with `require`: `node bin/kuzzle start --mappings ../somewhere/mappings.json`

Reply:
Makes sense.